### PR TITLE
added CategoryController and other functionality

### PR DESCRIPTION
**Pull request by ArtemReizer**

### CODE REVIEW

**Review by ohatsenko — APPROVED** [review]

File: src/main/java/com/app/bookstore/model/Category.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+@Data
+@Entity
+@Table(name = "categories")
+@SQLDelete(sql = "UPDATE categories SET is_deleted=true WHERE id=?")
+@Where(clause = "is_deleted=false")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
```
Comment (vovasalyha) [inline src/main/java/com/app/bookstore/model/Category.java]: Please replace it with @Column(nullable = false), they have one significant difference which is imprtant

File: src/main/resources/db/changelog/changes/09-insert-fiction-book.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            columns:
+              - column: {name: title, value: "1984"}
+              - column: {name: author, value: "George Orwell"}
+              - column: {name: isbn, value: 9780198185215}
+              - column: {name: price, value: 19.99}
+
+        - insert:
+            tableName: book_category
+            columns:
+              - column: {name: book_id, value: 1}
```
Comment (vovasalyha) [inline src/main/resources/db/changelog/changes/09-insert-fiction-book.yaml]: Better compute the value of id by ISBN for example, to elimitate the posibility that book with id 1 is absent, the same for category

**Review by vovasalyha — CHANGES_REQUESTED** [review]
GJ. Let's improve it a bit

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**